charts/osm/values.yaml

```diff
@@ -93,6 +93,9 @@ OpenServiceMesh:
   # -- Service certificate validity duration for certificate issued to workloads to communicate over mTLS
   serviceCertValidityDuration: 24h
 
+  # -- Certificate key bit size for data plane certificates issued to workloads to communicate over mTLS
```

Comment:
Maybe certificate manager too can go under the same object

Reply:
I think it would be a good idea to do something like:
```
certificateProvider:
  kind: <tresor|cert-manager.io|vault>
  serviceCertValiditityDuration: 24h
  rsaKeyBitSize: 4096
  ...
```

and map to the MeshConfig preset-mesh-config.yaml template.